packages/dashboard-core-plugins/package.json

```diff
@@ -32,7 +32,9 @@
     "@deephaven/grid": "file:../grid",
     "@deephaven/icons": "file:../icons",
     "@deephaven/iris-grid": "file:../iris-grid",
+    "@deephaven/jsapi-bootstrap": "file:../jsapi-bootstrap",
     "@deephaven/jsapi-shim": "file:../jsapi-shim",
```

Comment:
Removing this in a future update... looks like the only thing that will need to be updated is the `DropdownFilterPanel`